src/styles/components/status-bar/styles.less

```diff
@@ -99,6 +99,10 @@
         background-color: @yellow;
       }
 
+      &.color-7 {
```

Comment:
Would be nice if this class was `.color-8` to keep consistent with the relationship between color IDs and colors in the chart styles. However, I noticed there were other class names that had different colors in each file (e.g.: `.path-color-2` is pink here, but red in chart styles styles).

Are the colors intentionally different for color IDs between these two files? If so, why?

Reply:
Most probably not 🤷‍♂️